### PR TITLE
Fix all typography variants that cause warning at institution details

### PR DESCRIPTION
**Pull request by igorbrasileiro**
Also changed names of constants to UPPER_CASE - suggestion at #69  comments

### CODE REVIEW

File: src/components/InstitutionPage/DetailsPage/InstitutionInfo.js [hunk 1]
```diff
@@ -66,26 +66,26 @@ const InstitutionInfo = ({ classes, onHandleCreateEntryOpenToggle, institution }
         src="https://s3.amazonaws.com/tinycards/image/f8bda7d1c863b4f42adf2d1e5d72ff14" />
       <InstitutionInfos className={classes.institutionInfos}>
         <Typography
-          variant="title"
+          variant="h6"
           gutterBottom
         >
           {institution.name}
         </Typography>
         <Typography
```
Comment (Klynger) [inline src/components/InstitutionPage/DetailsPage/InstitutionInfo.js]: It should have a property component="span"

File: src/components/InstitutionPage/DetailsPage/InstitutionInfo.js [hunk 2]
```diff
@@ -66,26 +66,26 @@ const InstitutionInfo = ({ classes, onHandleCreateEntryOpenToggle, institution }
...
-          variant="subheading"
+          variant="subtitle1"
           gutterBottom
         >
           Email: {institution.email}
         </Typography>
         <Typography
```
Comment (Klynger) [inline src/components/InstitutionPage/DetailsPage/InstitutionInfo.js]: It should have a property component="span"

File: src/components/InstitutionPage/DetailsPage/InstitutionInfo.js (hunk 1 above)
Comment (igorbrasileiro) [reply src/components/InstitutionPage/DetailsPage/InstitutionInfo.js]: Fixed! @Klynger

File: src/components/InstitutionPage/DetailsPage/InstitutionInfo.js (hunk 2 above)
Comment (igorbrasileiro) [reply src/components/InstitutionPage/DetailsPage/InstitutionInfo.js]: Fixed! @Klynger

**Pull request merged**